app/views/phases/submissions.html.erb

```diff
@@ -6,5 +6,6 @@
     <p>This challenge phase does not currently have any submissions.</p>
   </div>
 <% else %>
+  <%= render partial: "submissions_stats", locals: { submissions: @submissions } %>
```

Comment:
don't think the locals are needed here (or below) since we already have the instance variable defined and available to the views